tests/test_wps_xclim_indices.py

```diff
@@ -44,6 +44,8 @@ def test_indicators_processes_discovery(indicator):
         parameters.remove("indexer")
         parameters.add("month")
         parameters.add("season")
+    if 'bootstrap' in parameters:
+        parameters.add("bootstrap")
```

Comment:
What is happening here? If it is in the set, add it to the set?

Reply:
Brain malfunction.